File: examples/java/time/src/main/java/ftl/time/Time.java
package ftl.time;

import java.time.OffsetDateTime;

import xyz.block.ftl.Export;
import xyz.block.ftl.Verb;

public class Time {

    @Verb
    @Export
    public TimeResponse time() {
        return new TimeResponse(OffsetDateTime.now(), "foo");
    }
}

File: examples/kotlin/time/src/main/kotlin/ftl/time/Time.kt
package ftl.time

import xyz.block.ftl.Export
import xyz.block.ftl.Verb
import java.time.OffsetDateTime

data class TimeResponse(val time: OffsetDateTime)

@Verb
@Export
fun time(): TimeResponse {
  return TimeResponse(time = OffsetDateTime.now())
}
